twoliter/embedded/imghelper

```diff
@@ -244,6 +282,18 @@ mkfs_data_xfs() {
   dd if="${bottlerocket_data}" of="${target}" conv=notrunc bs=1M seek="${offset}"
 }
 
+check_image_size() {
```

Comment:
I considered putting this in `partyplanner`, but if it lives here then we can use it in `generate_verity_root` without sourcing `partyplanner`.